pkg/v1/record/schemas.go

```diff
@@ -14,6 +14,9 @@ const (
 	TypeSOA     Type = "SOA"
 	TypeMX      Type = "MX"
 	TypeSRV     Type = "SRV"
+	TypeCAA     Type = "CAA"
+	TypeSSHFP   Type = "SSHFP"
+	TypeALIAS   Type = "ALIAS"
```

Comment:
It would be great if we add more examples with new types to `/pkg/v1/record/doc.go`